lib/components/molecules/Carousel/Carousel.js

```diff
@@ -0,0 +1,398 @@
+// @flow
+/**
+ * Carousel
+ */
+import React, { useContext, useState } from 'react';
+import styled from 'styled-components';
+import classnames from 'classnames';
+
+import stylesCarousel from './Carousel.style';
+import { Props, AnimationKeys } from './types';
+import { ListVanilla } from '../List';
+import Button from '../../atoms/Button';
+import useInterval from './useInterval.hook';
+
+const CarouselContext = React.createContext([]);
```

Comment:
Don't use context with-in the molecules.

Reply:
We agreed upon this.